news/5119.bugfix

```diff
@@ -0,0 +1 @@
+changed the icon of toolbar to gear icon.
```

Comment:
Always use proper English grammar and syntax, and give yourself credit in news items.

```suggestion
Changed the icon of toolbar to gear icon. @abhinaisai2002
```